install_config/aggregate_logging.adoc

```diff
@@ -441,11 +528,10 @@ own storage, but an {product-title} deployment configuration shares storage
 volumes between all its pods. So, when scaled up, the EFK deployer ensures each
 replica of Elasticsearch has its own deployment configuration.
 
-It is possible to scale your cluster up after creation by adding more
-deployments from a template; however, scaling up (or down) requires
-the correct procedure and an awareness of clustering parameters (to be
-described in a separate section). It is best to indicate the
-desired scale at first deployment.
+It is possible to scale your cluster up after creation by modifying the
+`openshift_logging_es_cluser_size` in the inventory file and re-running the
+logging playbook. Additional clustering parameters can be modified and are
+described in a separate section.
```

Comment:
With the "in a separate section" bit above, is it referring to the link below? Should that be closer to it or be an embedded link if so?